Yoast/ruleset.xml

```diff
@@ -2,6 +2,9 @@
 <ruleset name="Yoast" namespace="YoastCS\Yoast">
 	<description>Yoast Coding Standards</description>
 
+	<!-- Autoload the autoloaders of the dependencies. -->
```

Comment:
I've ran some tests, but I don't think this is actually needed as the ruleset includes the WPCS `WordPress` ruleset and that ruleset already loads the WPCS autoload. This can be (and should be) removed.